tests/admin-ui-tests/utils.ts

```diff
@@ -115,7 +124,7 @@ export const adminUITests = (
       beforeAll(async () => {
         await deleteAllData(projectDir);
       });
-      tests(playwright[browserName]);
+      tests(playwright[browserName], deleteAllData);
```

Comment:
pass `deleteAllData` to our test fn so we can call it as necessary from within each test file.